services/email.js

```diff
@@ -9,9 +9,11 @@ export default {
     name: 'email',
     // At least one of the CRUD methods is Required
     create: (req, resource, params, body, config, callback) => {
-        if (!params.from || !params.to || !params.subject || !params.message)
+        if (!params.subject || !params.message)
             return callback('Error: Missing parameters', null);
 
+        const to = SMTP.to, from = SMTP.from;
```

Comment:
I think you misunderstood me, or maybe I'm missing something here? It seems the person's email address gets lost along the way, so we have no way to know who they are or to contact them back ? I only said the `to` address should be a config parameter.
Also, I think we should avoid to simple names for local variables, like `to` or `from`, a better would toEmail, or toAddress and such ...